src/material-experimental/theming/_card.scss

```diff
@@ -0,0 +1,93 @@
+@use 'sass:color';
+@use 'sass:meta';
+@use '@angular/material' as mat;
+@use './token-resolution';
+
+// TODO(mmalerba): This should live under material/card when moving out of experimental.
+
+/// Gets tokens for setting the card's shape.
+/// @param {String} $shape The card's shape.
+/// @return {Map} A map of tokens for setting the card's shape.
+// Note: we use a function rather than simple rename, because we want to map a single shape value to
+// multiple tokens, rather than offer separate shape customizations for elevated and outlined cards.
+@function _get-tokens-for-card-shape($shape) {
+  @return (
+      (mdc, elevated-card): (container-shape: $shape),
```

Comment:
It's a little weird to me that we're using an array as a key here. If all it's doing is joining the values with a dash, maybe we can just write out `mdc-elevated-card`?

Reply:
We could look into doing that. I think it might be ok to combine it, but MDC does treat the prefix as two separate pieces. What I would want to avoid is needing to break up `mdc-elevated-card` into its separate prefix parts, since we can't just do a simple split on `-`. Its easier to go from pieces to whole than whole to pieces.